pkgs/desktops/gnome/extensions/guillotine/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitLab,
+  glib
```

Comment:
Add a leading comma to reduce diff in case new inputs will be added in the future:

```suggestion
  glib,
```